portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
```

Comment:
in the future, try to pull a discrete change like this into a separate commit, with a reflective commit message (the current commit message doesn't describe this)